benchmarks/benchmarks/data_objects.py

```diff
@@ -67,15 +67,5 @@ def time_slice(self):
     else:
 
         def time_slice(self):
-            slc = self.ds.h.slice(0, 0.5, self.field_name)
+            slc = self.ds.slice(0, 0.5, self.field_name)
             slc[self.field_name]
-
```

Comment:
I'm not saying we should keep the `benchmarks` directory.  But if we're going to, we should probably only excise the yt 2 stuff in this function.

Reply:
TBH I don't know what this code is supposed to be run and I didn't want to go to deep into this. Here I just made sure that this file wouldn't break horribly.
I'm not confident in updating the benchmark files further: looks to me like most of the if/else are written with only yt 2 VS yt 3 in mind, and the mere fact that we are now at 4.x is going to break them anyway.

Reply:
How about just remove benchmarks as part of this?

Reply:
Unless @Xarthisius ever wants to bring it back -- I think he was not convinced we gained much beyond what the test infrastructure brings.

Reply:
I'd be inclined to remove those in a separate PR, because I intend that this one can later serve as a model for how to prep a release on the specific aspect of deprecation :)

Reply:
It can go away as far as I'm concerned.

Reply:
Thanks, it's gone now !